rebasehelper/build_tools/koji_tool.py

```diff
@@ -90,8 +90,14 @@ def _scratch_build(cls, source, **kwargs):
         task_dict = cls.koji_helper.watch_koji_tasks(session, [task_id])
         task_list = []
         package_failed = False
+        build_error = None
         for key in six.iterkeys(task_dict):
             if task_dict[key] == koji.TASK_STATES['FAILED']:
+                try:
+                    # Gets the task results but the value is not used in this function currently
+                    info = session.getTaskResult(key)
```

Comment:
The unused variable is still there ;-)
And the comment doesn't really explain anything, what I had in mind is something like this:
```python
# call getTaskResult, as it raises an exception containing desired error message
```

Reply:
Alright, I thought you meant commenting, why the assignment is there. I get it now.